src/main/features/claim/routes/claim-receipt.ts

```diff
@@ -0,0 +1,43 @@
+import * as express from 'express'
+import { Paths } from 'claim/paths'
+import { ErrorHandling } from 'common/errorHandling'
+import { DocumentsClient } from 'app/documents/documentsClient'
+import * as http from 'http'
+import * as HttpStatus from 'http-status-codes'
+import { Claim } from 'claims/models/claim'
+
+const documentsClient: DocumentsClient = new DocumentsClient()
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(Paths.receiptReceiver.uri,
+    ErrorHandling.apply(
+      async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+
+        const { externalId } = req.params
+        const claim: Claim = res.locals.user.claim
```

Comment:
Please move it down near place it is really needed. You need it in `response.on('end', () => {` block.

Please also make sure you cannot access somebody's else claim, e.g situation that you are neither claimant or defendant on the case and you try to retrieve PDF. I believe we are covered by middleware but worth double checking.